packages/wrangler/src/__tests__/delete.test.ts

```diff
@@ -270,3 +435,48 @@ function mockListKVNamespacesRequest(...namespaces: KVNamespaceInfo[]) {
 		})
 	);
 }
+
+function mockListReferencesRequest(
+	forScript: string,
+	references: ServiceReferenceResponse = {}
+) {
+	msw.use(
+		rest.get(
+			"*/accounts/:accountId/workers/scripts/:scriptName/references",
```

Comment:
Are these new endpoints live and stable in EWC?

Reply:
they are indeed - these have been out and stable since pre-dev week, references for much longer